src/rust/src/tests/ed25519.rs

```diff
@@ -1,50 +1,735 @@
 use crate::librustzcash_zebra_crypto_sign_verify_detached;
 
+fn hex_vk(s: &str) -> [u8; 32] {
+    let mut tmp = [0u8; 32];
+    let mut l = 0;
+    for (i, c) in s.chars().enumerate() {
+        if i % 2 == 0 {
+            l = c.to_digit(16).unwrap();
+        } else {
+            tmp[i / 2] = (l << 4 | c.to_digit(16).unwrap()) as u8;
+        };
+    }
+    tmp
+}
+
+fn hex_sig(s: &str) -> [u8; 64] {
```

Comment:
`ed25519-zebra` already depends on the `hex` crate for debug output.  Do you want me to add a `FromStr` impl for these types?

Reply:
I don't remember why I didn't use the `hex!` macro here. I could switch to using that. I don't think a `FromStr` impl is necessary.

Reply:
I moved the test vectors to `test_consensus.cpp` so the ones in the Rust code were no longer necessary.